generator/generator_tests/osm_type_test.cpp

```diff
@@ -2151,14 +2151,10 @@ UNIT_CLASS_TEST(TestWithClassificator, OsmType_SimpleTypesSmoke)
     // {"landuse", "farmyard"},
     // {"landuse", "greenfield"},
     // {"landuse", "greenhouse_horticulture"},
-    // {"landuse", "retail"},
-    {"man_made", "cairn"},
+    // {"landuse", "retail"},    {"man_made", "cairn"},
```

Comment:
`{"man_made", "cairn"}` is re-added below (line 2498), no need to comment it here